OneSky.Services.Tests/Basic/Access/BasicAccessTests.cs

```diff
@@ -26,11 +28,16 @@ public void TestSatellitePasses()
             passRequest.FromObjectDark = true;
             passRequest.IncludePathCzml = true;
             passRequest.Verify();
+            var expected = JsonConvert.DeserializeObject<SatellitePassResults<ServiceCartographicWithTime>>(TestHelper.SatelliteAccessPassData);
 
             // call the service
             var passResults = AccessServices.GetSatellitePasses<SatellitePassResults<ServiceCartographicWithTime>>
                                                                                                 (passRequest).Result;
-            Assert.That(passResults.Passes.Count > 0);
+            Assert.That(passResults.Passes.Count == 18);
+
+            // object graph verification of actual results with expected results
+            passResults.Passes.Should().BeEquivalentTo(expected.Passes);
```

Comment:
Some of my tests are not passing because of small differences in double values.  Could you use Should().BeApproximately(..) with a specified precision of 1e-8 (for example) to eliminate those errors?

Reply:
Thanks for providing the error messages, I'll take a look at updating the precision level checks.